libs/sdk-core/src/breez_services.rs

```diff
@@ -807,7 +807,8 @@ impl BreezServices {
     /// Onchain receive swap API
     ///
     /// Create and start a new swap. A user-selected [OpeningFeeParams] can be optionally set in the argument.
-    /// If set, and the operation requires a new channel, the SDK will try to use the given fee params.
+    /// If set, and the operation requires a new channel, the SDK will enforce the use the given fee params.
```

Comment:
Nit: There's 2 times `the` in the sentence. Suggestion:
```suggestion
    /// If set, and the operation requires a new channel, the SDK will use the given fee params.
```